docs/querying/sql-data-types.md

```diff
@@ -34,12 +34,23 @@ Druid associates each column with a specific data type. This topic describes sup
 
 Druid natively supports the following basic column types:
 
-* LONG: 64-bit signed int
-* FLOAT: 32-bit float
-* DOUBLE: 64-bit float
-* STRING: UTF-8 encoded strings and string arrays
-* COMPLEX: non-standard data types, such as nested JSON, hyperUnique and approxHistogram, and DataSketches
-* ARRAY: arrays composed of any of these types
+* `LONG`: 64-bit signed int
+* `FLOAT`: 32-bit float
+* `DOUBLE`: 64-bit float
+* `STRING`: UTF-8 encoded strings and string arrays
+* `ARRAY`: arrays composed of any of these types
+
+## Complex types
+
+Druid natively supports the following complex types:
+* `COMPLEX<JSON>`: stores a copy of structured data in JSON format and specialized internal columns and indexes for nested basic types. Click here to learn more about [`COMPLEX<JSON>`](nested-columns.md)
+* `cardinality`: Data structure to compute the cardinality of Apache Druid dimensions using the HyperLogLog algorithm. Click here to learn more about [`cardinality`](hll-old.md#cardinality-aggregator)
+* `hyperUnique`: Data structure of aggregated values to estimate count distinct using a variant of the HyperLogLog approximation algorithm. Consider using HLL sketches for better accuracy in many cases. Click here to learn more about [`hyperUnique`](hll-old.md#hyperunique-aggregator)
```

Comment:
I'm not sure if we want to use these still. The filename is called old, and the page is not included in the nav

https://druid.apache.org/docs/latest/querying/hll-old

We also try to follow the Google style guide, so we don't say "Click here to learn more". It's typically

```
For more information (about XYZ), see PAGE/SECTION_TITLE(LINK)
```

`(about XYZ)` is optional depending on what the page/section title is

Reply:
We should mention the complex types for DataSketches and link to https://druid.apache.org/docs/latest/development/extensions-core/datasketches-extension/

Maybe something like this? "Complex types for sketches, including HLL and Theta sketch. For more information, see ..."